taxonomy-editor-frontend/src/pages/project/editentry/ListEntryChildren.tsx

```diff
@@ -131,7 +145,11 @@ const ListEntryChildren = ({ url, urlPrefix, setUpdateNodeChildren }) => {
             alignItems="center"
           >
             <Link
-              to={`${urlPrefix}/entry/${relationObject["child"]}`}
+              to={
+                hasChanges
+                  ? "#"
+                  : `${urlPrefix}/entry/${relationObject["child"]}`
+              }
```

Comment:
The Alert is a wonderful idea, but I don't think we should change the link: imagine a user wants to open an existing child node in a new tab with a ctrl+click or right-click+open in new tab.

I think we should only remove the link for newly created children.

Reply:
Created: https://github.com/openfoodfacts/taxonomy-editor/issues/474